packages/@react-aria/utils/src/scrollIntoView.ts

```diff
@@ -22,20 +30,25 @@ export function scrollIntoView(scrollView: HTMLElement, element: HTMLElement) {
   let height = element.offsetHeight;
   let x = scrollView.scrollLeft;
   let y = scrollView.scrollTop;
-  let maxX = x + scrollView.offsetWidth;
-  let maxY = y + scrollView.offsetHeight;
+
+  // Account for top/left border offsetting the scroll top/Left
+  let {borderTopWidth, borderLeftWidth} = window.getComputedStyle(scrollView);
```

Comment:
getComputedStyle is safe enough, we're already forcing layout with offsetHeight a couple lines up